EventTicketingSystem.ConsoleTest/Program.cs

```diff
@@ -0,0 +1,9 @@
+﻿using EventTicketingSystem.ConsoleTest;
+
+var seedService = ServiceProvider.GetSeedService();
```

Comment:
please use integratin testing for such propouses. Each repo shall be covered with all necessary tests

Reply:
I have created EventTicketingSystem.Tests project for integration testing